config/requirements.txt

```diff
@@ -1,13 +1,13 @@
 blinker==1.4
-Flask==1.1.2
+Flask==2.0.1
 Flask-Firehose==0.2.3
 Flask-Limiter==1.4
 Flask-SQLAlchemy==2.4.4
-Flask-WTF==0.14.3
+Flask-WTF==0.15.1
 GitHub-Flask==3.2.0
 Pillow==8.3.2
 python-dotenv==0.15.0
 requests==2.25.1
 ua-parser==0.10.0
-Werkzeug==1.0.1
+Werkzeug==2.0.1
```

Comment:
Ah interesting.
We added Werkzeug in the past because we needed to freeze it for CircleCI. 
https://github.com/webcompat/webcompat.com/commit/64ff3b05da1a47d9d282f1dc90ec35d8afa5c775

We do not use CircleCI anymore and we obviously stopped freezing the number, but we kept it in the requirements :) 

There are multiple strategies for requirements.txt in python.

1. Include all the submodules with version numbering which are imported by the main ones, so there is no surprise with unexpected breakages.
2. Trust the machine aka the installs by the main importer.

We have been using the 2. 
So we probably should be dropping Werkzeug here, as it will be installed by Flask. :)